service/test/BatchServerTest.cpp

```diff
@@ -1185,8 +1193,11 @@ TEST_F(BatchServerTest, fork_and_terminate_parent)
  * @test Enables coverage for the path of the SIGCHLD handler upon normal operation,
  *       when a forked child process terminates, and sends a SIGCHLD to the server.
  */
+
 TEST_F(BatchServerTest, action_sigchld)
```

Comment:
This doesn't resolve the issue with this test.  I've seen this hang on mcfly and other systems.

Reply:
Agreed.  I didn't mark this PR as "fixing" that issue.